src/index.js

```diff
@@ -179,10 +180,9 @@ const compare = (condition, value, attributes) => {
       return Array.isArray(value) && Array.isArray(compareValue) && value.every(x => compareValue.includes(x));
 
     case 'matches':
-      if (compareValue === undefined) return undefined;
       // we want to construct expression here
       // eslint-disable-next-line security/detect-non-literal-regexp
-      const regex = new RegExp(compareValue);
```

Comment:
Also will need to catch regex syntax error here, or else add regex syntax validation up front in `validate()` or `account-service`.

Reply:
invalid regex exceptions is still a problem here